specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/examples/SuccessfulSentimentRequest.json

```diff
@@ -0,0 +1,326 @@
+              "neutral": 0.0,
+              "negative": 0.0
+            },
+            "sentences": [
+              {
+                "text": "Great atmosphere.",
+                "sentiment": "positive",
+                "confidenceScores": {
+                  "positive": 1.0,
+                  "neutral": 0.0,
+                  "negative": 0.0
+                },
+                "offset": 0,
+                "length": 17,
+                "aspects": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 1,
+                      "negative": 0
+                    },
+                    "offset": 6,
+                    "length": 10,
+                    "text": "atmosphere",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/0/opinions/0"
+                      }
+                    ]
+                  }
+                ],
+                "opinions": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 1,
+                      "negative": 0
+                    },
+                    "offset": 0,
+                    "length": 5,
+                    "text": "great",
+                    "isNegated": false
+                  }
+                ]
+              },
+              {
+                "text": "Close to plenty of restaurants, hotels, and transit!",
+                "sentiment": "neutral",
+                "confidenceScores": {
+                  "positive": 0.13,
+                  "neutral": 0.86,
+                  "negative": 0.01
+                },
+                "offset": 18,
+                "length": 52,
+                "aspects": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 0.99,
+                      "negative": 0.01
+                    },
+                    "offset": 37,
+                    "length": 11,
+                    "text": "restaurants",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/1/opinions/0"
+                      }
+                    ]
+                  },
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 0.99,
+                      "negative": 0.01
+                    },
+                    "offset": 50,
+                    "length": 6,
+                    "text": "hotels",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/1/opinions/0"
```

Comment:
I think this should be `ref` or `$ref` (latter is the same like swagger references)

Reply:
I think the $ref is the format under schema or items in the Swagger definition and here we do naming the "links" itself. Since we are having the elements as the JSON pointer value itself. I think this is correct ??

 "default": {
            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
            "schema": {
              "$ref": "#/definitions/ErrorResponse"
            }